src/app/GitCommands/Settings/AppSettings.cs

```diff
@@ -1649,6 +1684,7 @@ public static void LoadSettings()
             {
                 // Set environment variable
                 GitSshHelpers.SetGitSshEnvironmentVariable(SshPath);
+                MigrateEditorSettings();
```

Comment:
We need to check the bitness - we'll only building and publishing x64 version (now?) but it's possible to build x86 (and arm64) version. With this, we only need to migrate if the current version isn't x86 version.


...or does it matter?

Reply:
If we build x86, the settings will be overwritten with x86 again, so no major issue.
For other than x64 this is should not be necessary, but should not matter.

So it is better to check. What should we check? Platform?
(and if you run this without installing the 4.3 alpha installer, the migration will be to early, but that can be seen as a feature.)

Reply:
no ifdef added now